src/command_modules/azure-cli-sql/tests/test_sql_commands.py

```diff
@@ -865,3 +866,84 @@ def test_sql_elastic_pools_mgmt(self, resource_group, resource_group_location, s
         self.cmd('sql elastic-pool delete -g {} --server {} --name {}'
                  .format(rg, server, self.pool_name),
                  checks=[NoneCheck()])
+
+
+class SqlServerImportExportMgmtScenarioTest(ScenarioTest):
+    @ResourceGroupPreparer()
+    @SqlServerPreparer()
+    @StorageAccountPreparer()
+    def test_sql_db_import_export_mgmt(self, resource_group, resource_group_location, server, storage_account):
+        location_long_name = 'West US'
+        admin_login = 'admin123'
+        admin_password = 'SecretPassword123'
+        db_name = 'cliautomationdb01'
+        db_name2 = 'cliautomationdb02'
+        container = 'bacpacs'
+
+        firewall_rule_1 = 'allowAllIps'
+        start_ip_address_1 = '0.0.0.0'
+        end_ip_address_1 = '255.255.255.255'
+
+        loc_long = location_long_name
+        rg = resource_group
+        sa = storage_account
```

Comment:
None blocking, but not recommended. Longer variable names improve the readability. Reassign and shorten the names defeat the purpose.